flask_mongoengine/wtf/fields.py

```diff
@@ -6,20 +6,24 @@
 import sys
 
 from wtforms import widgets
-from wtforms.fields import SelectFieldBase, TextAreaField, StringField
+from wtforms import fields as f
```

Comment:
Why?
It is nothing of consequence to import directly everything you want. It won't impact potential cycle imports because of how you are using these fields.
What is much worse, that is the one-letter variable.